rollup.config.js

```diff
@@ -0,0 +1,44 @@
+
+import babel from 'rollup-plugin-babel';
+import resolve from 'rollup-plugin-node-resolve';
+import uglify from 'rollup-plugin-uglify';
+import { minify } from 'uglify-es';
+const min = process.env.MINIFY;
+
+export default {
+    input: 'src/index.umd.js',
+    output: {
+      name: 'Select',
+      file: getDist(min),
+      format: 'umd',
+    },
+    globals: {
+      react: 'React',
+      'prop-types': 'PropTypes',
+      'react-dom': 'ReactDOM',
+      'react-input-autosize': 'AutosizeInput',
+      'classnames': 'classNames',
+    },
+    external: [
+      'classnames',
+      'prop-types',
+      'react-input-autosize',
+      'react',
+      'react-dom',
+    ],
+    plugins: [
+      babel({
+        babelrc: false,
+        exclude: 'node_modules/**',
+        presets: [['es2015', { 'modules': false }], 'stage-0', 'react'],
```

Comment:
Webpack 3 would benefit by this default as well. Maybe we can remove most of this babel config and keep it in the `babelrc` file?

Reply:
good point, I've fixed up the babel config to share common presets and plugins across the two configurations